sourcegraph/cody
# Command Hints: Add enrollment events for A/B test

## Description

Regardless of if a user is excluded or included in the ghost text experiment, this PR means that we will always fire an enrollment event.

We only fire this event one time per VS Code session, this means that we do not fire an unnecessarily large amount of telemetry logs (basically every time a user makes a selection)

## Test plan

1. Enable setting (either via settings or via A/B feature `cody-command-hints`)
2. Make a multi line selection (or a single line selection that includes all characters from the line)
3. Check correct enrollment is sent.

^ Do the same with the setting disabled.
